src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -171,7 +171,7 @@ public String verifyCodeword(@PathVariable("id") final Long id,
         FieldError error = new FieldError(
                 FORM_OBJECT_NAME,
                 GUESS_FIELD_NAME,
-                "Špatné řešení, zkuste se víc zamyslet :-)");
+                gameLogicService.getErrorBadSolutionMessage());
```

Comment:
Jo, to je lepší. Ale nevím, jestli bych to dával zrovna do gameLogic služby.
Buď na to vytvořil nějaou generickou Translation službu, která ti na základě klíče vrátí řetězec nebo ten MessageSource použív přímo tady v kontroleru.

GameLogic totiž spíš obsahuje logiku pro samotnou hru, nevím jestli bych do toho zatahoval ještě překlady. To už mi připadá trochu jako porušení Single responsibility Principle.